### PR TITLE
Fixes Organization Not Found in Left Drawer 

### LINKED ISSUE #1333: Organization not found error in left drawer
**Describe the bug**
When user profile is opened through the orglist page, an error is shown on the member page and redirected back to orglist page.

1. Login 
2. Click on the username on the bottom left

**Expected behavior**
No error should be shown on the member page and should not redirect back to orglist.

https://github.com/PalisadoesFoundation/talawa-admin/assets/123248648/be02ad60-17be-4c32-973a-a3bf5fe5b67e

### PR DESCRIPTION
**Pull request by GlenDsza**
**What kind of change does this PR introduce?**

bugfix

**Issue Number:**

Fixes #1333 

**Did you add tests for your changes?**

Yes

**Snapshots/Videos:**

https://github.com/PalisadoesFoundation/talawa-admin/assets/109027247/34a96370-4708-4fbd-a04b-72f7d5e2528e

**Summary**

After Login, Opening User profile generated no organization found error, and user profile opened through 'My Organizations' page used to render '`LeftDrawerOrg`' which is meant to be rendered if user profile is opened from 'organization dashboard'.

- The issue is fixed by conditionally rendering `LeftDrawerOrg` Component if UserProfile is reached from organization dashboard and rendering `LeftDrawer` Component if UserProfile reached from 'My Organizations Page' 
- Added basic CSS to highlight User Profile Option in Left Drawer if User Profile Page is open.
 
- Refactored (Simplified/Efficient) `OrganizationScreen` & `SuperAdminScreen` and their respective tests.

**Does this PR introduce a breaking change?**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by Cioppolo14** [comment]
@GlenDsza Please fix the failing test application check.

**Comment by palisadoes** [comment]
Please increase the code coverage of your patch so that the tests pass

**Comment by palisadoes** [comment]
_**NOTE**_ Read very carefully

1. We just merged this PR which upgraded the prettier package. 
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/1599
1. It reformatted over 150 files. 

This will put your PR at risk of extensive merge conflicts. Do the following **IN THIS ORDER**:

1. upgrade your `prettier` in your local branch to the same version
1. run `prettier` on your local branch
1. update your local branch with the latest upstream
    
This will help to reduce the number of future merge conflicts for your PR.

**Comment by GlenDsza** [comment]
Hello @Cioppolo14  , I think the PR is ready for review.

**Comment by GlenDsza** [comment]
I had a suggestion, 
- the [`SuperAdminScreen.tsx`](https://github.com/PalisadoesFoundation/talawa-admin/blob/develop/src/components/SuperAdminScreen/SuperAdminScreen.tsx) and [`OrganizationScreen.tsx`](https://github.com/PalisadoesFoundation/talawa-admin/blob/develop/src/components/OrganizationScreen/OrganizationScreen.tsx) both act as container component for `LeftDrawer.tsx` and `LeftDrawerOrg.tsx` respectively 
- but most lines of code are similar and serve the same purpose with minimal difference. 
- Maybe, we can merge both the files in one file (maybe called `leftDrawerContainer`) and rest handle the difference using minimal conditional logic

Actually I have worked already on this thing, if y'all agree should I Add the changes to this PR only or should create new issue and then create PR for it

File: src/components/LeftDrawer/LeftDrawer.module.css [hunk 1]
```diff
@@ -60,9 +60,9 @@
 .leftDrawer .profileContainer {
   border: none;
   width: 100%;
+  padding: 2.1rem 0.5rem;
   height: 52px;
   border-radius: 8px;
-  background-color: var(--bs-white);
```
Comment (beingnoble03) [inline src/components/LeftDrawer/LeftDrawer.module.css:65]: Why is this changed?

File: src/components/LeftDrawer/LeftDrawer.tsx [hunk 2]
```diff
@@ -104,34 +104,43 @@ const leftDrawer = ({
...
+              className={styles.profileContainer}
+              variant={screenName === 'Profile' ? 'success' : 'light'}
+              data-testid="profileBtn"
+            >
+              <div className={styles.imageContainer}>
+                {userImage && userImage !== 'null' ? (
+                  <img src={userImage} alt={`profile picture`} />
+                ) : (
+                  <img
+                    src={`https://api.dicebear.com/5.x/initials/svg?seed=${firstName}%20${lastName}`}
```
Comment (beingnoble03) [inline src/components/LeftDrawer/LeftDrawer.tsx:124]: Same comment as in `src/components/LeftDrawerOrg/LeftDrawerOrg.tsx`

**Review by beingnoble03 — CHANGES_REQUESTED** [review]
please address the comments. rest lgtm.

File: src/components/LeftDrawerOrg/LeftDrawerOrg.tsx [hunk 3]
```diff
@@ -176,33 +165,43 @@ const leftDrawerOrg = ({
...
+              variant={screenName === 'Profile' ? 'success' : 'light'}
+              className={styles.profileContainer}
+              data-testid="profileBtn"
+            >
+              <div className={styles.imageContainer}>
+                {userImage && userImage !== 'null' ? (
+                  <img src={userImage} alt={`profile picture`} />
+                ) : (
+                  <img
+                    src={`https://api.dicebear.com/5.x/initials/svg?seed=${firstName}%20${lastName}`}
```
[the remark this answers is not in the document]
Comment (GlenDsza) [reply src/components/LeftDrawerOrg/LeftDrawerOrg.tsx:185]: Sure👍

File: src/components/LeftDrawer/LeftDrawer.module.css (hunk 1 above)
Comment (GlenDsza) [reply src/components/LeftDrawer/LeftDrawer.module.css:65]: The above changes were made to match styling of profileContainer from `LeftDrawerOrg`  (similar component for org screen)
https://github.com/GlenDsza/talawa-admin/blob/030647b0a2511daa43b185e5a8d4c9aedf990280/src/components/LeftDrawerOrg/LeftDrawerOrg.module.css#L86C1-L95C2

The resulting grey background highlights the profileContainer from drawer white background
image

File: package-lock.json [hunk 4]
```diff
@@ -5926,12 +5926,6 @@
         "node": ">=8"
       }
     },
-    "node_modules/ansi-sequence-parser": {
-      "version": "1.1.1",
-      "resolved": "https://registry.npmjs.org/ansi-sequence-parser/-/ansi-sequence-parser-1.1.1.tgz",
-      "integrity": "sha512-vJXt3yiaUL4UU546s3rPXlsry/RnM730G1+HkpKE012AN0sx1eOrxSu95oKDIonskeLTijMgqWZ3uDEe3NFvyg==",
-      "peer": true
-    },
     "node_modules/ansi-styles": {
```
Comment (rishav-jha-mech) [inline package-lock.json:5929]: @GlenDsza revert any change you make to this file

File: README.md [hunk 5]
```diff
@@ -25,7 +25,7 @@ Core features include:
 <!-- toc -->
 
 - [Talawa Components](#talawa-components)
-- [Documentation](#documentation)
+- [Documentation -](#documentation--)
```
Comment (rishav-jha-mech) [inline README.md:28]: Revert any change you make to this file

Comment (GlenDsza) [reply README.md:28]: Following changes were automatically added to the commit due the scripts running during commit 
and if i revert them once again while committing same change would be added.


https://github.com/PalisadoesFoundation/talawa-admin/assets/109027247/b0891956-311e-479d-a6f2-5422c893f3f0

had created the issue for same: #1624

**Comment by palisadoes** [comment]
Please fix the conflicting files

File: package-lock.json (hunk 4 above)
Comment (GlenDsza) [reply package-lock.json]: Done

File: README.md (hunk 5 above)
Comment (GlenDsza) [reply README.md]: As the upstream (PalisadoesFoundation/develop) contains the above in README hence resolving

**Comment by GlenDsza** [comment]
@rishav-jha-mech @beingnoble03 Have resolved conflicts, added tests and  addressed comments, need review

**Comment by Olatade** [comment]
@GlenDsza please can you update your branch with the latest changes.

File: src/components/LeftDrawer/LeftDrawer.module.css (hunk 1 above)
Comment (beingnoble03) [reply src/components/LeftDrawer/LeftDrawer.module.css:65]: Ah. Ohk.

**Review by beingnoble03 — APPROVED** [review]
nice work. lgtm.

**Comment by palisadoes** [comment]
@GlenDsza 

Your PR has too many unnecessary files that don’t relate to your issue.

 REDAME.md

You can from the PR by running the commands below from the root directory of the repository

git add -u
git reset HEAD path/to/file
git commit

**Comment by GlenDsza** [comment]
@palisadoes @beingnoble03 Removed `README.md` from changed files, need approving review

**Review by palisadoes — APPROVED** [review]
